qiskit/transpiler/coupling.py

```diff
@@ -334,15 +335,17 @@ def draw(self):
             PIL.Image: Drawn coupling map.
 
         Raises:
-            ImportError: when pydot or pillow are not installed.
+            MissingOptionalLibraryError: when pydot or pillow are not installed.
         """
 
         try:
             import pydot
             from PIL import Image
         except ImportError as ex:
-            raise ImportError(
-                "CouplingMap.draw requires pydot and pillow. " "Run 'pip install pydot pillow'."
+            raise MissingOptionalLibraryError(
+                libname="pillow",
+                name="coupling map drawer",
+                pip_install="pip install pillow",
```

Comment:
Should this be 
```suggestion
                pip_install="pip install pydot pillow",
```

Reply:
Good catch! I unfold that one in eb4de93780d90400df0f07c6d4548e295247a289